src/lib/utils.js

```diff
@@ -22,40 +20,6 @@ function getPairs (arr) {
   }, [])
 }
 
-/**
- * Deterministically generate a UUID from a secret and a public input.
```

Comment:
Right now Mojaloop depends on having the same transfer ID across all of the ledgers. Taking the transfer id out of the LPI will make it more difficult to upgrade the connector used in that project. Do we have any alternative way of attaching the transfer ID from the previous ledger and making the connector use the same one on the outgoing transfer, or do we need to just make Mojaloop not depend on having the same transfer IDs?